src/common/ManagedCommon/ModuleType.cs

```diff
@@ -30,5 +30,6 @@ public enum ModuleType
         MeasureTool,
         ShortcutGuide,
         PowerOCR,
+        New,
```

Comment:
The same as on the policy name. This is confusing. Please rename to `NewPlus`.

Reply:
> The same as on the policy name. This is confusing. Please rename to `NewPlus`.

Thank you! I've finally gotten around to renaming to NewPlus throughout. Thanks again.